CONTRIBUTING.md

```diff
@@ -1,66 +1,35 @@
+If you are running GitHub Desktop, you can clone this repository locally from
+GitHub using the "Clone in Desktop" button from the Octokit.net project page,
+or run this command in your Git-enabled shell:
 
-`git clone git@github.com:octokit/Octokit.net.git Octokit`
+`git clone https://github.com/octokit/Octokit.net.git Octokit`
 
-If you want to make contributions to the project, 
-[forking the project](https://help.github.com/articles/fork-a-repo) is the 
+If you want to make contributions to the project,
+[forking the project](https://help.github.com/articles/fork-a-repo) is the
 easiest way to do this. You can then clone down your fork instead:
 
-`git clone git@github.com:MY-USERNAME-HERE/Octokit.net.git Octokit`
-
-After doing that, run the `.\build.cmd` script at the root of the repository 
-to ensure all the tests pass.
-
-### How is the codebase organised?
-
-The two main projects are the `Octokit` and `Octokit.Reactive` projects.
-
-The `Octokit.Reactive` library is a thin wrapper over the `Octokit` 
-library - for those who want to use Reactive Extensions (Rx) instead of tasks.
-
-The namespaces are organised so that the relevant components are easy to discover:
+`git clone https://github.com/MY-USERNAME-HERE/Octokit.net.git Octokit`
 
- - **Authentication** - everything related to authenticating requests 
- - **Clients** - the logic for interacting with various parts of the GitHub API
- - **Exceptions** - types which represent exceptional behaviour from the API 
- - **Helpers** - assorted extensions and helpers to keep the code neat and tidy
- - **Http** - the internal networking components which Octokit requires
- - **Models** - types which represent request/response objects
+After doing that, run the `.\build` script at the root of the repository
+to ensure everything builds and the tests pass.
 
-Unless you're modifying some core behaviour, the **Clients** and **Models** namespaces
-are likely to be the most interesting areas.
+### How can I get involved?
 
-The clients within a project are organized similarly to the endpoints in the
-[GitHub API documentation](http://developer.github.com/v3/)
-
-Some clients are "sub-clients". For example, when you navigate to the
-[Issues API](http://developer.github.com/v3/issues/) you'll notice there's an
-endpoint for issues. But in the right navbar, there are other APIs such as
-[Assignees](http://developer.github.com/v3/issues/assignees/) and
-[Milestones](http://developer.github.com/v3/issues/milestones/).
-
-We've tried to mirror this structure. So the `IObservableMilestoneClient` isn't
-a direct property of `IObservableGitHubClient`. Instead, it's a property of the
-`IObservableIssuesClient`. And thus you can get to it by going to
-`client.Issues.Milestones`.
-
-### What needs to be done?
-
-We have a [`easy-fix`](https://github.com/octokit/octokit.net/issues?labels=easy-fix&state=open)
+We have a [`up-for-grabs`](https://github.com/octokit/octokit.net/issues/labels/up-for-grabs)
```

Comment:
a**n**  'up-for-grabs' label ("an" rather than "a")?

Also this link goes to the issue search page which is including all up-for-grabs items both opened and closed.  Should it only be for opened?  
https://github.com/octokit/octokit.net/labels/up-for-grabs